src/qml/filters/alpha_adjust/ui.qml

```diff
@@ -15,36 +15,37 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import QtQml.Models 2.12
 import QtQuick 2.12
 import QtQuick.Controls 2.12
 import QtQuick.Layouts 1.12
 import Shotcut.Controls 1.0 as Shotcut
-import QtQml.Models 2.12
 
 Item {
-    width: 200
-    height: 50
     property string paramOperation: '2'
     property string paramThreshold: '3'
     property string paramAmount: '4'
     property string paramInvert: '5'
 
+    width: 200
+    height: 50
     Component.onCompleted: {
         if (filter.isNew) {
-            filter.set(paramOperation, 0)
-            filter.set(paramThreshold, 0.5)
-            filter.set(paramAmount, 0.5)
+            filter.set(paramOperation, 0);
```

Comment:
Looking through the changes, I like most things. The change that would require the most adjustment for me is adding semicolons after function calls.